.gitignore

```diff
@@ -14,3 +14,8 @@
 /releases
 /build
 /lib
+plugin/info.xml
```

Comment:
Why are these files added to the .gitignore file? 

The updates to info.xml and plugin.xml will need to made too.

Reply:
We don't need anything in the .gitignore file. We will fix the info.xml and plugin. xml and get back to you